sdk/openai/openai/sources/generated/src/api/operations.ts

```diff
@@ -189,6 +424,9 @@ export async function _getCompletionsDeserialize(
                   severity: p.content_filter_results?.self_harm?.["severity"],
                   filtered: p.content_filter_results?.self_harm?.["filtered"],
                 },
+            error: !p.content_filter_results?.error
```

Comment:
I remember we did some custom stuff for parsing error content types, are that stuff needed now that we have this in code gen?

Reply:
I'll double check this

Reply:
We still need the `deserializeContentFilter` function for the customized method. Our current API separate the `ContentFilterResults` into success versus error result, so I'll update the implementation to use the generated output like above instead.